.github/workflows/lint.yml

```
@@ -228,6 +228,16 @@ jobs:
           pip install torch --pre --index-url https://download.pytorch.org/whl/nightly/cpu/
       - name: Run run_test.py (nonretryable)
         run: |
+          set -ex
+
+          pushd /tmp
+          TORCH_PATH=$(python -c 'import torch; import os; import inspect; print(os.path.dirname(inspect.getfile(torch)))')
```

TIL that one can do `inspect.getfile(foo)` instead of `foo.__file__`, what's the benefit?
```suggestion
          TORCH_PATH=$(python -c 'import torch; import os; print(os.path.dirname(torch.__file__))')
```